hal/hal_remote/include/hal_remote/hal_binary_decoder.h

```diff
@@ -93,7 +93,7 @@ class hal_binary_decoder {
       case hal_binary_encoder::COMMAND::DEVICE_DELETE_REPLY:
         return sizeof(uint32_t);  // bool as 32 bit
       default:
-        fprintf(stderr, "Unknown command : %d\n", (int)command);
+        (void)fprintf(stderr, "Unknown command : %d\n", (int)command);
```

Comment:
Can we either use `std::cerr` everywhere, or use `fprintf` everywhere? There are some annoying subtleties when you mix the two. I *think* that won't affect us, but I'm not 100% sure and if we can avoid having to think about that, that will make things easier.

Reply:
yes fair enough, will change